### DISCUSSION TITLE
24-hour clock

### DISCUSSION THREAD

**Discussion by DieSteinhose** [krateng/maloja#82] [Ideas]
I really enjoy the program, I wish I'd gotten to know it sooner.

I just have a little trouble reading the timestamps. Here is a 24 hours system normal and not an AM/PM system.

Is there a way to implement this easily?

**Comment by krateng** [comment]
Implemented in d3e45b1

Just run the server once, then check the default settings for the option `TIME_FORMAT`

**Reply by janaxhell** [reply]
Three years later this seems not valid anymore. The Time Format entry in the settings UI shows an empty dialog and the wiki has no explanation, just that [TIME_FORMAT](https://github.com/krateng/maloja/blob/master/settings.md) is Time Format. What do I enter in the dialog to set it to 24hrs?

**Reply by duckcantcode** [reply]
> Three years later this seems not valid anymore. The Time Format entry in the settings UI shows an empty dialog and the wiki has no explanation, just that [TIME_FORMAT](https://github.com/krateng/maloja/blob/master/settings.md) is Time Format. What do I enter in the dialog to set it to 24hrs?

@janaxhell Check out https://strftime.org/ it looks like that's the format you'll follow. In #226 I linked my discord server for further help

**Reply by janaxhell** [reply]
Perfect! I used this string `%a %d %b %Y %H:%M `
Thanks
